docs/production-deployment/self-hosted-guide/checklist.mdx

```diff
@@ -46,106 +77,81 @@ At a high level, you will want to track these 3 categories of metrics:
 
-The fundamental recommendation here is to use [Task Queue–based versioning](#task-queuebased-versioning) whenever you can, and use the [Patch and GetVersion APIs](#patch-and-getversion-apis) if your Workflows live long enough that you cannot afford to wait for them to complete before changing their behavior.
+## Maintenance and Upgrades 
 
-### Task Queue–based versioning
+Temporal Server releases as often as every two weeks.
+Temporal recommends [upgrading sequentially](/self-hosted-guide/upgrade-server), not skipping any minor versions.
+It is acceptable to skip patch versions.
+Temporal recommends keeping up-to-date and not falling behind on your server versions.
+No support is guaranteed for Temporal Server, but very old servers will be hard for even the community to support, so we encourage you to keep up to date. 
+In addition to the core service, infrastructure to run Temporal, such as Kubernetes to host the service and data stores for persistence, has to be implemented and maintained.
 
-In this approach, after you make changes to your Workflow code, deploy your Workers pointed at a new Task Queue instead of the existing one.
-For example, if you were targeting the Task Queue `tq-v1`, you'd now target `tq-v2` or similar.
-You must also update whatever sources are starting new Workflows to point at the new Task Queue name.
-Leave your old Workers running, possibly reducing the number of instances, until no more open Workflows are on `tq-v1`.
-Then you can decommission all of them.
+Temporal service availability can be affected by server upgrades.
+We recommend load and availability testing during the upgrade process to understand the performance envelope.
 
-**Advantages**
+Temporal Cloud updates are managed by the Temporal Cloud team.
+Cloud upgrades are seamless.
 
-- Conceptually simple.
-- Robust.
-  Changes are isolated from each other in a way that makes mistakes unlikely.
+## Expert Support
 
-**Disadvantages**
+Temporal recommends that customer platform teams who are building out a Temporal service gain deep experience across the lifecycle and breadth of a Temporal application.
 
-- Operationally complex (need to keep old Workers alive and change the Task Queue that clients point to).
-  This approach can result in multiple sets of old Workers, especially with long-running Workflows.
-- Can't be used to fix a bug in currently running or open Workflows.
+Specific activities include:
+- [Worker tuning](/develop/worker-performance)
+- [Worker best practices](/workers)
+- Code reviews
+- Design guidance
+- Training
+- Code reviews
+- Security reviews
+- [Metrics](/references/sdk-metrics) and monitoring
+- Technical onboarding
 
-### Patch and GetVersion APIs
+Temporal support provides guidance on all of the above.
+Temporal support will work with your teams to make this easier and support teams that support Temporal.
```

Comment:
The word support is used 3 times in this sentence. Also, whenever mentioning Temporal support, I normally add a link to [file a ticket with Temporal Support] and work it in there somehow, so it's right at the point where people might be when they're trying to work through a solution

Reply:
Good point -- I'm just going to remove this redundant sentence, I think adding a link to the Support page to the previous sentence is sufficient.